src/k-provider/ovp/provider.js

```diff
@@ -49,7 +49,7 @@ export default class OVPProvider extends BaseProvider<OVPProviderMediaInfoObject
       const entryId = mediaInfo.entryId;
       const referenceId = mediaInfo.referenceId;
       if (entryId || referenceId) {
-        let ks: string = this.ks;
+        let ks: string = this.ks || this.anonymousKS;
```

Comment:
need to do the same here - https://github.com/kaltura/playkit-js-providers/blob/34fb6af2037831b917a132c2c8c8c59a489cf5f7/src/k-provider/ovp/provider.js#L47 ?